Sources/Quick/NSBundle+CurrentTestBundle.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+extension NSBundle {
+
+    /**
+     Locates the first bundle with a '.xctest' file extension.
```

Comment:
Hmm, is it possible to have multiple test bundles? What happens then? Based on the code below that uses `NSBundle.currentTestBundle`, I'd bet that if a test case class isn't in the first test bundle, this method would return `nil`, even if the test case exists in another bundle.

At first I thought Quick's own tests may have two test bundles:

<img width="670" alt="screen shot 2016-01-16 at 11 01 55 am" src="https://cloud.githubusercontent.com/assets/552921/12373939/9cc03426-bc40-11e5-8c45-11bd24a054b5.png">

But inspecting at runtime from within `QuickFocused-iOSTests` shows that they're not both available:

```
(lldb) po NSBundle.allBundles()
▿ 4 elements
  - [0] : NSBundle </Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/usr/lib> (not yet loaded)
  - [1] : NSBundle </Users/bgesiak/Library/Developer/Xcode/DerivedData/Quick-druxpcbupaazldhgnyterfnxgtal/Build/Products/Debug-iphonesimulator/QuickFocused-iOSTests.xctest> (loaded)
  - [2] : NSBundle </Users/bgesiak/Library/Developer/Xcode/DerivedData/Quick-druxpcbupaazldhgnyterfnxgtal/Build/Products/Debug-iphonesimulator/QuickFocused-iOSTests.xctest/Frameworks> (not yet loaded)
  - [3] : NSBundle </Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Xcode/Agents> (loaded)
```

So I'm assuming this will work fine. Still, it would be nice to know if it's possible to have two test bundles at once. Any ideas?